src/pages/VouchingPage/NewNominationModal.tsx

```diff
@@ -58,70 +40,116 @@ export const NewNominationModal = ({
           : ''
       }`
     : '';
+  type NominateFormValues = {
+    name: string;
+    address: string;
+    description: string;
+  };
+  const intialValues = {
+    name: '',
+    address: '',
+    description: '',
+  };
 
-  const isAddress = ethers.utils.isAddress(address);
-  const nominateChanged =
-    name.length == 0 || description.length == 0 || isAddress;
-
-  const onChangeWith =
-    (set: (v: string) => void) => (e: React.ChangeEvent<HTMLInputElement>) =>
-      set(e.target.value);
+  const {
+    control,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<NominateFormValues>({ mode: 'all' });
 
-  const onSubmit = async () => {
-    if (!nominateChanged) {
-      throw 'Submit called when form not ready.';
-    }
+  const onSubmit: SubmitHandler<NominateFormValues> = async data => {
     nominateUser({
-      name,
-      address,
-      description,
+      ...data,
     })
       .then(() => {
         onClose();
       })
       .catch(console.warn);
   };
-
   return (
-    <FormModal
-      title="Nominate New Member"
-      submitDisabled={!nominateChanged}
-      onSubmit={onSubmit}
-      submitText="Nominate Member"
-      open={visible}
-      onClose={onClose}
-      size="small"
-    >
-      <p className={classes.description}>{nominateDescription}</p>
-      <div className={classes.quadGrid}>
-        <ApeTextField
-          label="Name"
-          value={name}
-          onChange={onChangeWith(setName)}
-          fullWidth
-        />
-        <ApeTextField
-          label="ETH Address"
-          value={address}
-          onChange={onChangeWith(setAddress)}
-          error={address.length > 0 && !isAddress}
-          fullWidth
-        />
-        <ApeTextField
-          label="Why are you nominating this person?"
-          placeholder="Tell us why the person should be added to the circle, such as what they have achieved or what they will do in the future."
-          value={description}
-          className={classes.gridAllColumns}
-          onChange={onChangeWith(setDescription)}
-          multiline
-          rows={4}
-          inputProps={{
-            maxLength: 280,
-          }}
-          fullWidth
-        />
-      </div>
-    </FormModal>
+    <Modal title="Nominate New Member" open={visible} onClose={onClose}>
+      <Form nominate onSubmit={handleSubmit(onSubmit)}>
+        <Text>{nominateDescription}</Text>
```

Comment:
Please add `lineHeight: 'normal'` to the styles here so that the text does not look so crowded